about/introduction.rst

```diff
@@ -6,105 +6,98 @@ Introduction
-  the engine development, e.g. how to report bugs, help with the documentation, etc.
-  It also contains subsections intended for advanced users and contributors
-  to the engine development, with the information on compiling the engine,
-  contributing to the editor, or developing C++ modules.
-- The **Community** section is dedicated to the life of its community.
-  It points to various community channels like Godot Contributors Chat and
-  Discord and contains a list of recommended third-party tutorials outside
-  of this documentation.
-- Finally, the **Class reference** is the documentation of the Godot API,
-  which is also available directly within the engine's script editor. It is
-  generated automatically from a file in the main source repository, therefore
-  the generated files of the documentation are not meant to be modified. See
-  :ref:`doc_updating_the_class_reference` for details.
+- **Contributing** gives information related to contributing to
+  Godot, whether to the core engine, documentation, demos or other parts,
+  e.g. how to report bugs, organization and processes, etc.
+  It also contains sections intended for advanced users and contributors,
+  with information on compiling the engine, contributing to the editor,
+  or developing C++ modules.
+- **Community** dedicated to the life of Godot's community.
+  It points to various community channels like the Godot Contributors Chat and
+  Discord and contains a list of recommended third-party tutorials and
+  materials outside of this documentation.
+- Finally, the **Class reference** documents the full Godot API,
+  also available directly within the engine's script editor.
+  You can find information on all classes, functions, signals and so on here.
 
 In addition to this documentation, you may also want to take a look at the
 various `Godot demo projects <https://github.com/godotengine/godot-demo-projects>`_.
 
-Have fun reading and making games with Godot Engine!
+About this documentation
+------------------------
+
+Members of the Godot Engine community continuously write, correct, edit, and
+improve this documentation. We are always looking for more help. You can also
+contribute by opening Github issues or translating the documentation into your language.
+If you are interested in helping, see :ref:`Ways to contribute <doc_ways_to_contribute>`
+and :ref:`Writing documentation <doc_contributing_writing_documentation>`,
+or get in touch with the `Documentation team <https://godotengine.org/teams/#documentation>`_
+on `Godot Contributors Chat <https://chat.godotengine.org/>`_.
+
+All documentation content is licensed under the permissive Creative Commons Attribution 3.0
+(`CC BY 3.0 <https://creativecommons.org/licenses/by/3.0/>`_) license,
+with attribution to "*Juan Linietsky, Ariel Manzur, and the Godot Engine community*"
+unless otherwise noted.
+
+*Have fun reading and making games with Godot Engine!*
```

Comment:
The whole *About this documentation* paragraph got moved to the end as its a bit self-serving and not the most important information for a brand new user. At the same time, I've made the plea for contributors/help more direct and added links on how to get in contact, join and contribute instead of needless details about Sphinx or RST.